Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 group :test do
   #gem 'turbolinks'
   gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'ab1d8d1a4a1af748ae7009d9d725193d780d0059'
+  gem 'rails-settings-cached', github: 'merqlove/rails-settings-cached', branch: 'master'
```

Comment:
without merge fix https://github.com/huacnlee/rails-settings-cached/pull/110  in original repo we can merge this pr